core/src/main/clojure/xtdb/tx_producer.clj

```diff
@@ -91,10 +91,12 @@
 
 (def ^:private nullable-inst-type [:union #{:null [:timestamp-tz :micro "UTC"]}])
 
+
 (def ^:private ^org.apache.arrow.vector.types.pojo.Field tx-ops-field
   (types/->field "tx-ops" (ArrowType$Union. UnionMode/Dense (int-array (range 6))) false
-                 (types/col-type->field 'sql [:struct {:query :utf8
-                                                       :params [:union #{:null :varbinary}]}])
+                 (types/col-type->field 'sql [:struct {"query" :utf8
+                                                       "params" [:union #{:null :varbinary}]}])
```

Comment:
This is a case where `name` to `str` posed problems at for keywords they differ.

Reply:
right, yep - I think we try to use symbols for these anyway, so `'[:struct {query :utf8, params [...]}]`